manifests/m/Mathpix/MathpixSnippingTool/03.00.0074/Mathpix.MathpixSnippingTool.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Mathpix.MathpixSnippingTool
+PackageVersion: 03.00.0074
+PackageLocale: en-US
+Publisher: Mathpix
+PackageName: Mathpix Snipping Tool
+License: Copyright (c) Mathpix
+ShortDescription: Mathpix Snipping Tool Setup

```

Comment:
Please provide a better short description of what Mathpix Snipping Tool is

Reply:
Done.